src/Server/AspNetCore.Playground/Resources/index.html

```diff
@@ -520,15 +520,15 @@
     </div>
   </div>
 
-  <div id="root"></div>
+  <div id="root" />
   <script type="text/javascript">
     window.addEventListener('load', function (event) {
 
-      var loadingWrapper = document.getElementById('loading-wrapper');
+      const loadingWrapper = document.getElementById('loading-wrapper');
```

Comment:
We should still use `var` here because it's `ES5` compliant. For example, if someone still uses `IE10` for whatever reason, this would result in an error.

Reply:
fixed